### PR TITLE
Modify travis script and re-structure tests dir

### PR DESCRIPTION
**Pull request by shravan97**
Features in this PR : 
* Modify travis script to install pyspark
* A tiny modification in the part of the script that runs the tutorials (combined `exit 1` with python)
* Move current integration tests to `integration_tests_local`

### CODE REVIEW

File: .travis-script.sh [hunk 1]
```diff
@@ -1,48 +1,56 @@
-sudo apt-get --yes update
-sudo apt-get --yes install python-pip
-sudo pip install enum34
-sudo pip install nose
+# All apt-get installs
+sudo apt-get --yes update # Update
```
Comment (etejedor) [inline .travis-script.sh]: Are the comments of the next 4 lines a bit redundant, perhaps?

File: .travis-script.sh [hunk 2]
```diff
@@ -1,48 +1,56 @@
...
+sudo apt-get --yes install python-pip # Installing pip
+sudo apt-get --yes install wget # Installing wget
+sudo apt-get --yes install default-jdk # Install Java JDK
+
+# Install required packages
+sudo pip install -r /app/requirements.txt
```
Comment (etejedor) [inline .travis-script.sh]: What does this install?

Comment (etejedor) [reply .travis-script.sh]: Ok I see, enum34 and nose, should this change be in a separate commit?

Comment (shravan97) [reply .travis-script.sh]: @etejedor , Yup. That'd be better. I'll make a different commit for that 👍

File: .travis-script.sh (hunk 1 above)
Comment (shravan97) [reply .travis-script.sh]: Yup. They seem some what redundant. Shall I remove them then ?

**Comment by etejedor** [comment]
I would call `tests/integration_tests_local` just `tests/local`

File: .travis-script.sh (hunk 1 above)
Comment (etejedor) [reply .travis-script.sh]: Yes please

**Comment by shravan97** [comment]
@etejedor , but we're separating only the integration tests right ?

**Comment by etejedor** [comment]
Then I would have:
- `tests/integration` (and `tests/integration/local`)
- `tests/unit`

**Comment by shravan97** [comment]
Sure @etejedor . That sounds good. Will make that change 👍

File: .travis-script.sh (hunk 2 above)
Comment (shravan97) [reply .travis-script.sh]: @etejedor , any other changes ? If not, I'll rebase commits into 3, including a commit for this one (`requirements.txt`)

**Comment by etejedor** [comment]
Please go ahead @shravan97

**Comment by shravan97** [comment]
@etejedor done now 👍

File: .travis-script.sh [hunk 3]
```diff
@@ -1,48 +1,57 @@
-sudo apt-get --yes update
+# All apt-get installs
+sudo apt-get --yes update # Update
```
Comment (etejedor) [inline .travis-script.sh]: Still an unnecessary comment here

**Comment by etejedor** [comment]
Do we need to update the script that runs the tests as well, given that we are updating the paths of the tests?

**Comment by shravan97** [comment]
I changed it @etejedor 👍

File: .travis-script.sh (hunk 3 above)
Comment (shravan97) [reply .travis-script.sh]: That comment is just to distinguish the installs from the update. I can remove that as well if you insist

**Comment by shravan97** [comment]
@etejedor , did you mean changing the script for tests in `.travis-script.sh` ?

**Comment by etejedor** [comment]
I meant this:
https://github.com/shravan97/PyRDF/blob/master/run_tests.sh

File: .travis-script.sh (hunk 3 above)
Comment (etejedor) [reply .travis-script.sh]: Yes thanks

**Comment by shravan97** [comment]
Yes @etejedor . That's right. I forgot to stage that. Will push a new commit for that 👍

**Comment by shravan97** [comment]
Done @etejedor 👍

**Comment by shravan97** [comment]
@etejedor sorry ignore. I'll remove that comment and rebase again 👍

**Comment by shravan97** [comment]
Done now @etejedor 👍

**Pull request merged**